src/main/java/com/scaleset/cfbuilder/rds/DBInstance.java

```diff
@@ -0,0 +1,31 @@
+package com.scaleset.cfbuilder.rds;
+
+import com.scaleset.cfbuilder.annotations.Type;
+import com.scaleset.cfbuilder.core.Taggable;
+import com.scaleset.cfbuilder.ec2.SecurityGroup;
+
+@Type("AWS::RDS::DBInstance")
+public interface DBInstance extends Taggable {
+
+    DBInstance engine(Object engine);
```

Comment:
Can the parameter be different things than String?

Reply:
You're right, currently it it can't really be anything but a String. I'll change it.

The reason behind this is that I tried to be closer to the definition in other classes, but since we don't really check what engines are allowed we don't need "Object engine" and Strings will suffice.